src/app/app.component.ts

```diff
@@ -95,6 +98,16 @@ export class AppComponent implements OnInit {
     //   });
     // }
 
+    this.route.queryParams.subscribe(params => {
+      // TODO: make constants
```

Comment:
can drop this comment now?

Reply:
I think I forgot to make those query parameters constants. I'll add them to app.settings.ts.